oar/core/advisory_mgr.py

```diff
@@ -44,3 +78,20 @@ def __init__(self, **kwargs):
             super().__init__(**kwargs)
         except ErrataException as e:
             raise AdvisoryException("initialize erratum failed") from e
+
+    def change_qe_email(self, email):
+        """
+        Change advisory owner
+
+        Args:
+            email (str): owner's email address
+        """
+        self.update(qe_email=email)
+        self.commit()
+        logger.info(f"qe email of advisory {self.errata_id} is updated to {email}")
```

Comment:
how about  info(f"QA Owner of  advisory {self.errata_id} is update to {email}")?

Reply:
done